### PR TITLE
Fix payday emails

### PR DESCRIPTION
**Pull request by rohitpaulk**
#3503

### CODE REVIEW

File: gratipay/billing/payday.py [hunk 1]
```diff
@@ -542,32 +542,39 @@ def notify_participants(self):
...
+                                     AND subscriber = %(username)s
+                                ORDER BY team, mtime DESC
+                                ) s
+                           JOIN teams t ON s.team = t.slug
+                           JOIN participants p ON t.owner = p.username
+                          WHERE s.amount > 0
+                            AND t.is_approved IS true
+                            AND t.is_closed IS NOT true
+                            AND (SELECT count(*)
+                                   FROM exchange_routes er
```
Comment (whit537) [inline gratipay/billing/payday.py]: Probably should get in the habit of using `current_exchange_routes` here.

**Comment by whit537** [comment]
LGTM. Clear to merge when Travis is green, @rohitpaulk.

**Comment by whit537** [comment]
Assuming you like my commits. :)

**Comment by rohitpaulk** [comment]
Going to merge once Travis is green.

**Pull request merged**